codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/BoringSSLCertificateCallback.java

```diff
@@ -102,8 +102,7 @@ long[] handle(long ssl, byte[] keyTypeBytes, byte[][] asn1DerEncodedPrincipals,
 
         try {
             if (keyManager == null) {
-                engineMap.remove(ssl);
-                return null;
+                return new long[] { 0, 0 };
```

Comment:
What if there is a keyManager but it doesn't contain applicable valid key for the requested issuers (selectKeyMaterialClientSide below returning null)?

Reply:
Good catch... fixing as we speak